src/Tribe/Templates.php

```diff
@@ -679,7 +679,25 @@ public static function event_date_to_pubDate( $time, $d, $gmt ) {
 			global $post;
 
 			if ( is_object( $post ) && $post->post_type == Tribe__Events__Main::POSTTYPE && is_feed() && $gmt ) {
-				$time = tribe_get_start_date( $post->ID, false, $d );
+
+				// Don't interfere if the timezone hasn't been set
+				$zone = get_option( 'timezone_string', false );
+				if ( false === $zone ) {
+					return $time;
```

Comment:
Are we 100% sure that `$time` is in the desired format (`$d`) at this point in time? It may be a good plan to format the date (as UTC) rather than just returning here.